truss/truss_config.py

```diff
@@ -171,6 +172,98 @@ class ModelServer(Enum):
     TRT_LLM = "TRT_LLM"
 
 
+class TRTLLMModelArchitecture(Enum):
```

Comment:
maybe we can put these in a separate file for readability. This file is getting massive. maybe `truss/config/trt_llm.py`